mpd/client.go

```diff
@@ -360,6 +360,31 @@ func (c *Client) PlaylistInfo(start, end int) ([]Attrs, error) {
 	return cmd.AttrsList("file")
 }
 
+// Prio set the priority of the specified songs. If end is negative but start
+// is positive, it does it for the song at position start. If both start and
```

Comment:
Non-actionable comment: >= 0 mean non-negative, not "positive". I realize the same wording is used in other functions, so don't worry about it.